app/src/main/java/com/android/sample/ui/swipePage/SwipePage.kt

```diff
@@ -803,3 +797,33 @@ fun LikeDislikeIconAnimation(
         }
   }
 }
+
+@Composable
+fun RecipeImage(
```

Comment:
This function is made for the swipe page, I think it should be made private. I know it's means you need to test it through the Swipe page, but otherwise people might see RecipeImage and think it can be used more generally.

Reply:
I’d like to test the composable directly( Which I can't do if the composable is private if im not mistaken), so I passed an `isConnected` parameter to the function. This allows me to easily test scenarios where the user is offline, as simulating a lack of connectivity directly can be quite complex without using such an approach. 

Another option could be to move this function to a utility class, as it feels fairly generic. What are your thoughts on this approach?

Reply:
I think it would be better to put it in utils if it can be used eslewhere